webextensions/api/webRequest.json

```diff
@@ -1405,6 +1405,52 @@
               },
               "safari_ios": "mirror"
             }
+          },
+          "options": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "edge": "mirror",
+                "firefox": [
+                  {
+                    "version_added": "130",
+                    "notes": "Optional"
+                  },
+                  {
+                    "version_added": "62",
+                    "version_removed": "130",
+                    "notes": "Required"
+                  }
+                ],
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "requestId": {
```

Comment:
Let's drop `requestId` option here. It is a required part of the method. The method is meaningless without the ability to specify a requestId to identify the request.